content/content-strategy/cms/optimise-cms/section2.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: text/textblock
+---
+A CMS allows content teams to work together on content. With this in mind, you can set up various user roles for your web team in your CMS. Sometimes these roles may be the responsibility of one or more people.
+
+Each of these roles will have a different purpose and access level. For example, a person whose role is to proofread or edit, may not be able to publish content. This ensures quality control with clear processes.
+Some typical CMS roles include:
+- author: sometimes called a content creator or writer. This is the role of actually preparing and writing content within the CMS itself.
+- editor: edits existing content
+- approver: has the final say on whether to publish a piece of content
+- publisher: pushes content to the ‘live’ site
+
+### [1]Action
+Write down who is responsible for each part of the [content lifecycle](/content-strategy/content-lifecycle/), and what they will be required to do in the CMS.
```

Comment:
No page at `/content-strategy/content-lifecycle` is this coming in a later release?